src/merkle.ts

```diff
@@ -14,6 +14,7 @@ export type WitnessProof = {
 // Used to create a full binary tree.
 function padLeaves(hashes: Hash[]) {
   const paddingLength = Math.pow(2, Math.ceil(Math.log2(hashes.length))) - hashes.length;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can we add a lint rule so that variables that start with `_` are not flagged as unused? We should have a rule like that in the monorepo.

Reply:
3778528